### ISSUE TITLE
Fix for color texture green border issue

### ISSUE THREAD

**Issue by ceylina** [tullamods/OmniCC#154]
This happens currently and it is an easy fix:

![wowscrnshot_080616_131407](https://cloud.githubusercontent.com/assets/9342192/17458503/f77cae3a-5bd7-11e6-9ac1-e7db076f33d7.jpg)

**1)**
In OmniCC_Config\widgets\colorSelector.lua line 51

change
bg:SetTexture(1, 1, 1)
to
bg:SetColorTexture(1, 1, 1)

**2)**
In OmniCC_Config\widgets\fontSelector.lua lines - 87, 92,94, and 176

replace SetTexture with SetColorTexture

(example self.bg:SetTexture(1, 1, 1, 0.3) becomes self.bg:SetColorTexture(1, 1, 1, 0.3))

**3)**
In OmniCC_Config\widgets\radioGroup.lua line 51 and 55

replace SetTexture with SetColorTexture
---

I am unsure if you even need to do these
OmniCC_Config\widgets\groupEditor.lua line 236
bg:SetTexture(0, 0, 0, 0.5)

OmniCC_Config\widgets\listEditor.lua line 222
bg:SetTexture(0, 0, 0, 0.5)

but might want to to be safe?

Basically if you use SetTexture  for a color and not an image, it is now SetColorTexture

image textures like

thumb:SetTexture([[Interface\Buttons\UI-ScrollBar-Knob]])

stay the same

**Comment by Jaliborc** [comment]
Good catch!
